Userland/Libraries/LibJS/Runtime/Intl/Locale.cpp

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include <LibJS/Runtime/Array.h>
```

Comment:
Let's bump the copyright year :)
(same below)

Reply:
Done!